src/era_5g_relay_network_application/era_5g_relay_network_application/client.py

```diff
@@ -220,6 +220,7 @@ def tf_callback(transforms: List[TransformStamped]) -> None:
             data=extract_values(TFMessage(transforms=transforms)),
             topic_name="/tf",
             topic_type="tf2_msgs/TFMessage",
```

Comment:
Not a change, but should this be `tf2_msgs/msg/TFMessage`?

Reply:
true